dev-guide/api-conventions.md

```diff
@@ -109,6 +109,40 @@ This means, that any end user fetching the resource from the API can observe tha
 This has the effect of helping our users to understand how they might be able to configure their cluster, without
 having to search for features or review API schemas within the product docs.
 
+### Only one phrasing for each idea
+
+Each idea should have a single possible expression in the API structures, without having multiple ways to say the same thing.
+From [PEP 20][pep-20]:
+
+> * There should be one-- and preferably only one --obvious way to do it.
+> * Although that way may not be obvious at first unless you're Dutch.
+
+For example, [ClusterVersion's `spec.desiredUpdate`][desiredUpdate] is a pointer (because it was added before [the anti-pointer guidelines](#pointers)).
```

Comment:
I don't really like the wording here, "before the anti-pointer guidelines". It doesn't feel like the right tone of voice for an API guidelines doc. We are trying for passive voice and I think this is too active voice.

I think it would also be good to explain that it allows to representations of the same thing.

I'd suggest rewording the example to something like
```suggestion
For example, the [ClusterVersion's `spec.desiredUpdate`][desiredUpdate] is a pointer field.
This does not meet our current API guidelines (see our [pointer guidelines](#pointers)) as we advise against the use of pointers in most cases, but, in this example, it also allows the same semantic to be expressed in two different ways.

Both `nil`:
...
and the empty-struct:
...
represent the same "I do not desire an update" idea.
```

Reply:
After this is reworded, the update looks good to me.

Reply:
Rebased onto master and rolled in @JoelSpeed 's suggestions with a3f21140 -> f2d90a07.